fv3core/utils/future_stencil.py

```diff
@@ -355,15 +355,25 @@ def _load_stencil(self, stencil_id: int) -> Callable:
 
         return stencil_class
 
+    def _load_cached_stencil(self):
+        stencil_class: Callable = None
+        self._delay()
+
+        # try/except block to prevent loading incomplete files...
+        try:
+            stencil_class = self._builder.backend.load()
+        except (AttributeError, FileNotFoundError):
```

Comment:
Is this a sufficient condition to ensure loading completely written stencils? I guess so, since if the file exists it's hopefully complete, or does GT4Py write the file in multiple steps? Also, not entirely sure why `AttributeError` has to be here. Maybe a comment?

Reply:
Comment added!